packages/api/endpoints/granules.js

```diff
@@ -678,6 +680,140 @@ const associateExecution = async (req, res) => {
   });
 };
 
+const ApiFileSchema = z.object({
```

Comment:
Nit: consider this being something like ApiFileInputSchema - we already have schemas defined in too many places (e.g. DB typing, API TS typing, schema validation in lib/schemas.js)